rm-datasource/src/main/java/io/seata/rm/datasource/undo/oracle/OracleUndoInsertExecutor.java

```diff
@@ -47,9 +47,9 @@ protected String buildUndoSQL() {
             throw new ShouldNeverHappenException("Invalid UNDO LOG");
         }
         Row row = afterImageRows.get(0);
-        StringBuffer mainSQL = new StringBuffer("DELETE FROM " + keywordChecker.checkAndReplace(sqlUndoLog.getTableName()));
-        StringBuffer where = new StringBuffer(" WHERE ");
-        boolean first = true;
+        StringBuilder mainSQL = new StringBuilder("DELETE FROM " + keywordChecker.checkAndReplace(sqlUndoLog.getTableName()));
```

Comment:
append

Reply:
I can't get your point？Use append to replace +？

Reply:
yes